roles/hammerdb/templates/db_workload.yml.j2

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: hammerdb
-        #image: "quay.io/cloud-bulldozer/hammerdb:latest"
+        #image: "quay.io/cloud-bulldozer/hammerdb:master"
```

Comment:
same as above, can this be moved back to cloud-bulldozer's image?

Reply:
Uhm, one of the many clueful contributors to hammerd in snafu made a stupid mistake, so we need another PR for snafu first... Since there's code freeze for snafu I haven't created that PR yet.

Reply:
Can you please explain this stupid mistake ? maybe a link to the PR? and don't worry about code-freeze, we'll get your PR in.